### DISCUSSION TITLE
Best way to organise a tree of content?

### DISCUSSION THREAD

**Discussion by limegorilla** [directus/directus#10044] [Archive]
Bringing this question over from #5546 

What I've got is a documentation system - something like this

```
documentation/
  | - Top level subdirectory/
  |    | - Example doc
  |    | - Example subsubdirectory/
  |    |    | - Example doc
  | - Example doc
```

What I was trying to do is use a separate collection for the names of the parents - obviously then came up with the error that 'The tree view interface only works for recursive relationships'

Is this the best way to organise this? In my head it's a bit complex for end users as there is no separation in the back end, but it is recursive (something I do want) and allows for a child to have multiple parents, and for each parent to have it's own page.
However, I'm just not as sure how this will work in the front end, and I would rather get the structure correct before I try that :)

**Comment by rijkvanzanten** [comment]
Bringing this answer over from https://github.com/directus/directus/discussions/5546#discussioncomment-1868758

There's no way to achieve that in the current tree-view o2m interface. That one only works recursively on a single collection. You could achieve this with 2 collections with a simple m2o in between, but that wouldn't let you browse the items in a tree view currently
